environments/sandbox.yml

```diff
@@ -172,6 +172,10 @@ cname:
     ttl: 300
     record: "sdshmcts-sbox.azurefd.net"
     shutter: false
+  - name: "toffee2"
+    ttl: 300
+    record: "sdshmcts-sbox.azurefd.net"
```

Comment:
this should be your new frontdoor instance not an existing one

Reply:
updated with the new frontdoor